### PR TITLE
Fix for missing symbol occurrences in synthetics implicit views (cf. #1749)

### PR DESCRIPTION
**Pull request by fikovnik**

### CODE REVIEW

File: semanticdb/integration/src/main/scala/example/Synthetic.scala [hunk 1]
```diff
@@ -44,4 +44,10 @@ class Synthetic {
     if a < b
   } yield a
 
+  // See https://github.com/scalameta/scalameta/issues/1749
+  import scala.math.Ordered.orderingToOrdered
+  val x1 = new F
+  val x2 = new F
+  (x1, x1)
```
Comment (olafurpg) [inline semanticdb/integration/src/main/scala/example/Synthetic.scala]: Please `./bin/scalafmt --diff` to make CI green.

File: semanticdb/integration/src/main/scala/example/Synthetic.scala [hunk 2]
```diff
@@ -44,4 +44,10 @@ class Synthetic {
     if a < b
   } yield a
 
+  // See https://github.com/scalameta/scalameta/issues/1749
+  import scala.math.Ordered.orderingToOrdered
```
Comment (olafurpg) [inline semanticdb/integration/src/main/scala/example/Synthetic.scala]: Just curious, can this import and `new F` be replaced with `val x = 42; (x, x)`? Would be nice to minimize the test case as much as possible

**Comment by fikovnik** [comment]
I could do the example without import, but in that case I would have to define more terms (unless you can think of a better way) which will congest the example. I think that in this form it is fairly obvious what the intention is and it is visible from the `metac.expect` what has been missing.

File: semanticdb/scalac/library/src/main/scala/scala/meta/internal/semanticdb/scalac/TextDocumentOps.scala [hunk 3]
```diff
@@ -460,7 +460,7 @@ trait TextDocumentOps { self: SemanticdbOps =>
                 case gimpl: g.ApplyToImplicitArgs =>
                   gimpl.fun match {
                     case gview: g.ApplyImplicitView =>
-                      isVisited += gview
+                      isVisitedParent += gview
```
Comment (olafurpg) [inline semanticdb/scalac/library/src/main/scala/scala/meta/internal/semanticdb/scalac/TextDocumentOps.scala]: Impressive you tracked this down 😍

**Pull request merged**